svthalia/concrexit
# Optimize sales app

Might help with #2469 

### Summary
Rewrites some ugly code that also does quite some speedup. Also does some minor admin UI improvements. Hopefully this is enough of a speedup.

### How to test
1. Go to the shift admin page 
2. Check that it isn't that slow
